symengine/real_double.h

```diff
@@ -228,6 +228,9 @@ class RealDouble : public Number
      * */
     RCP<const Number> mulreal(const Integer &other) const
     {
+        if (other.is_zero()) {
```

Comment:
```suggestion
        if (other.is_exact_zero()) {
```

Reply:
We already know it is exact since it is `Integer` so `is_zero` should be enough.

Reply:
Yes, you are right.